deeplake/core/vectorstore/test_deeplake_vectorstore.py

```diff
@@ -184,12 +189,13 @@ def test_search_basic(local_path, hub_cloud_dev_token):
         read_only=True,
         token=hub_cloud_dev_token,
     )
+    assert vector_store_cloud.exec_option == "compute_engine"
```

Comment:
The idea is to pick the best option given the dataset storage, which is Indra for Wasabi datasets. 

I mentioned in slack that we should change the requirement in LangChain to deeplake[enterprise], i order to make sure people have the right packages.

Are there any other issues you see? I don't see why people would be confused, since the choice is made automatically.